### PR TITLE
Fix enterprise individual nemesis features missing

### PR DESCRIPTION
**Pull request by fgelcer**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
testing one LDAP test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/ldap/)
testing one SLA test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/sla/)

**Comment by fruch** [comment]
> > testing one LDAP test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/ldap/) testing one SLA test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/sla/)
> 
> the LDAP one finished with failure, but the nemesis run as expected, but the SLA one was still skipped... need to dig in and find out why. @fruch did I miss something that needs to be enabled?

for now it's enabled only on SLA test cases, 

https://github.com/scylladb/scylla-cluster-tests/pull/5868 introduce a configuration that enables them.

so you need to add `sla: true` to the configuration yaml, at least for the time being, until this would be test in longer/heavier cases, and this would be removed

File: test-cases/nemesis/longevity-5gb-1h-SlaIncreaseSharesByAttachAnotherSlDuringLoad.yaml [hunk 1]
```diff
@@ -18,6 +18,13 @@ azure_instance_type_db: 'Standard_L8s_v3'
 instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
+sla: true
```
Comment (fgelcer) [inline test-cases/nemesis/longevity-5gb-1h-SlaIncreaseSharesByAttachAnotherSlDuringLoad.yaml:21]: @fruch , i added it here

File: test-cases/nemesis/longevity-5gb-1h-SlaIncreaseSharesDuringLoad.yaml [hunk 2]
```diff
@@ -18,6 +18,13 @@ azure_instance_type_db: 'Standard_L8s_v3'
 instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
+sla: true
```
Comment (fgelcer) [inline test-cases/nemesis/longevity-5gb-1h-SlaIncreaseSharesDuringLoad.yaml:21]: and here

File: test-cases/nemesis/longevity-5gb-1h-SlaReplaceUsingDetachDuringLoad.yaml [hunk 3]
```diff
@@ -18,6 +18,13 @@ azure_instance_type_db: 'Standard_L8s_v3'
 instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
+sla: true
```
Comment (fgelcer) [inline test-cases/nemesis/longevity-5gb-1h-SlaReplaceUsingDetachDuringLoad.yaml:21]: and here

File: test-cases/nemesis/longevity-5gb-1h-SlaReplaceUsingDropDuringLoad.yaml [hunk 4]
```diff
@@ -18,6 +18,13 @@ azure_instance_type_db: 'Standard_L8s_v3'
 instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
+sla: true
```
Comment (fgelcer) [inline test-cases/nemesis/longevity-5gb-1h-SlaReplaceUsingDropDuringLoad.yaml:21]: and here

File: test-cases/nemesis/longevity-5gb-1h-SlaSevenSlWithMaxSharesDuringLoad.yaml [hunk 5]
```diff
@@ -18,6 +18,13 @@ azure_instance_type_db: 'Standard_L8s_v3'
 instance_type_loader: 'c5.large'
 azure_instance_type_loader: 'Standard_F2s_v2'
 
+sla: true
```
Comment (fgelcer) [inline test-cases/nemesis/longevity-5gb-1h-SlaSevenSlWithMaxSharesDuringLoad.yaml:21]: and here

**Comment by fgelcer** [comment]
> > > testing one LDAP test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/ldap/) testing one SLA test [here](https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/individual_nemesis/job/sla/)
> > 
> > 
> > the LDAP one finished with failure, but the nemesis run as expected, but the SLA one was still skipped... need to dig in and find out why. @fruch did I miss something that needs to be enabled?
> 
> for now it's enabled only on SLA test cases,
> 
> #5868 introduce a configuration that enables them.
> 
> so you need to add `sla: true` to the configuration yaml, at least for the time being, until this would be test in longer/heavier cases, and this would be removed

my mistake, i missed the only file i used to run the job above...
i just fixed it, and re-running

**Review by k0machi — APPROVED** [review]
LGTM. Bear in mind the next time someone generates aws configs this will get overriden - will need to think of a way to pass this information to the generator.

**Review by roydahan — APPROVED** [review]

**Comment by fgelcer** [comment]
@scylladb/qa-maintainers , this one needs to be backported

**Pull request merged**